backend/grant/proposal/views.py

```diff
@@ -218,11 +217,10 @@ def get_proposal_drafts():
 
 @blueprint.route("/<proposal_id>", methods=["PUT"])
 @requires_team_member_auth
-# TODO add gaurd (minimum, maximum, shape)
 @body({
     "title": fields.Str(required=True),
     "brief": fields.Str(required=True),
-    "category": fields.Str(required=True, validate=validate.OneOf(choices=Category.list())),
+    "category": fields.Str(required=True, validate=validate.OneOf(choices=[e for e in Category.list()] + [''])),
```

Comment:
I'm pretty sure python supports array arithmetic, just `Category.list() + ['']` should suffice